commands/operator-sdk/cmd/new.go

```diff
@@ -121,9 +123,10 @@ func mustBeNewProject() {
 
 func doScaffold() {
 	cfg := &input.Config{
-		Repo:           filepath.Join(projutil.CheckAndGetProjectGoPkg(), projectName),
-		AbsProjectPath: filepath.Join(projutil.MustGetwd(), projectName),
-		ProjectName:    projectName,
+		Repo:            filepath.Join(projutil.CheckAndGetProjectGoPkg(), projectName),
+		AbsProjectPath:  filepath.Join(projutil.MustGetwd(), projectName),
+		ProjectName:     projectName,
+		IsClusterScoped: isClusterScoped,
```

Comment:
`isClusterScoped` is specific to Roles and RoleBindings so it shouldn't go in `input.Config` or `scaffold.Scaffold`. I suggest adding an `IsClusterScoped` field and setting it in both `scaffold.Role` and `scaffold.RoleBinding` [here](https://github.com/operator-framework/operator-sdk/blob/master/commands/operator-sdk/cmd/new.go#L134-L135).